plugins/api-docs/src/components/ApisCards/ProvidedApisCard.tsx

```diff
@@ -71,11 +72,15 @@ export const ProvidedApisCard = ({ variant = 'gridItem' }: Props) => {
       title="Provided APIs"
       variant={variant}
       emptyContent={
-        <div>
-          No component provides this API.{' '}
-          <Link to="https://backstage.io/docs/features/software-catalog/descriptor-format#specprovidesapis-optional">
-            Learn how to provide APIs.
-          </Link>
+        <div style={{ textAlign: 'center' }}>
+          <Typography variant="h6">
+            This {entity.kind.toLowerCase()} does not provide its own APIs.
```

Comment:
Actually is still think that we should remove the term "own" here and stay with "any"

Reply:
Yep would agree with that

Reply:
Haven't really had a lot of time to look at this, so took your feedback here and applied it. Updated screen caps in PR description.